src/GafferSceneUI/SceneView.cpp

```diff
@@ -1335,77 +1335,21 @@ void SceneView::frame( const GafferScene::PathMatcher &filter, const Imath::V3f
 
 void SceneView::expandSelection( size_t depth )
 {
-	Context::Scope scopedContext( getContext() );
```

Comment:
Addressed in 263cb8314ab381821b5b803763e6904f5eef23a7, db4373d1998014e10ebedd8799beb868e66f8a35, and f6f343dcaae6bf8adcee5b1ad11ddb6969f7d1c7

Side note: As of f6f343dcaae6bf8adcee5b1ad11ddb6969f7d1c7, the docs seem to be building correctly now (e.g I _think_ SceneHierarchy expansion matches to master in the screengrabs).